### PR TITLE
Add charset utf-8 to restore

### PR DESCRIPTION
**Pull request by benrudolph**
@snopoke http://manage.dimagi.com/default.asp?235608 this fixes the unicode issue you were seeing in formplayer. the request library that we use in formplayer automatically encodes the response based on the response headers. since the charset wasn't defined, it must've been defaulting to some other charset. label to do some testing on the phone

cc: @dannyroberts

### CODE REVIEW

**Comment by benrudolph** [comment]
also @wpride

File: corehq/ex-submodules/casexml/apps/phone/restore.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def restore_cache_key(prefix, user_id, version=None):
 
 def stream_response(payload, headers=None, status=200):
     try:
-        response = StreamingHttpResponse(FileWrapper(payload), content_type="text/xml", status=status)
+        response = StreamingHttpResponse(FileWrapper(payload), content_type="text/xml; charset=utf-8", status=status)
```
Comment (dimagimon) [inline corehq/ex-submodules/casexml/apps/phone/restore.py]: E501 line too long (117 > 115 characters)

**Comment by czue** [comment]
assume label is on to test with phones?

**Comment by benrudolph** [comment]
yep

**Comment by benrudolph** [comment]
did a couple of restores on the phone and nothing broke. think this is fairly safe since this is how we want phones to interpret it anyways

**Comment by wpride** [comment]
cool. @phillipm this definitely seems like a good candidate for the integration tests - is there a way we can proactively run those on an un-merged branch? Or possible we can merge to master and then immediately run?

**Comment by benrudolph** [comment]
it's on staging if you can point the integration tests there

**Comment by phillipm** [comment]
Yeah, you'll probably want to make an app on staging that has the ability to restore utf-8 chars into the case db, the fixture db, the username, and verify that those chars load correctly during form entry. An initial login restore should suffice, so you don't need to use API endpoints to update data and do subsequent restores.
After you do all that you can just test it by hand or I can turn it into an integration test that runs off staging until this is deployed, at which time we can recreate the app on prod and point the test to that.

**Comment by benrudolph** [comment]
ok @phillipm i've tested a unicode fixture and case. i didn't test a unicode username cause i didn't know how to input it into the login easily. i did however have a unicode first and last name. photos from testing:

unicode fixture:
photo on 8-31-16 at 3 16 pm

unicode case:
photo on 8-31-16 at 3 17 pm

**Comment by snopoke** [comment]
Great poses there Ben!

**Comment by czue** [comment]
:hand: :phone: :angry:

**Comment by benrudolph** [comment]
this is g2g! @ctsims and @phillipm have signed off as well

**Pull request merged**